D2/Api/ApiChannel.qml

```diff
@@ -8,10 +8,50 @@ Item {
 
   readonly property string    pathPrefix:  "app.traktor.mixer.channels." + index + "."
 
-  AppProperty { id: propVolume;             path: pathPrefix + "volume";               onValueChanged: updateOnAirState() }
-  AppProperty { id: propXfaderAssignLeft;   path: pathPrefix + "xfader_assign.left";   onValueChanged: updateOnAirState() }
-  AppProperty { id: propXfaderAssignRight;  path: pathPrefix + "xfader_assign.right";  onValueChanged: updateOnAirState() }
-  AppProperty { id: propXfaderAdjust;       path: "app.traktor.mixer.xfader.adjust";   onValueChanged: updateOnAirState() }
+  AppProperty {
+    id: propVolume
+    path: pathPrefix + "volume"
+
+    onValueChanged: {
+      updateOnAirState()
+      ApiClient.send("updateChannel/" + index, {
+        volume: propVolume.value,
+      })
+    }
+  }
+  AppProperty {
+    id: propXfaderAssignLeft
+    path: pathPrefix + "xfader_assign.left"
+    
+    onValueChanged: {
+      updateOnAirState()
+      ApiClient.send("updateChannel/" + index, {
+        XfaderAssignLeft: propXfaderAssignLeft.value,
```

Comment:
Keys should start with a lowercase letter, so `xfaderAssignLeft`. Same for the `Xfader` keys below.